src/main/java/ng/abdlquadri/eventbus/EventBus.java

```diff
@@ -232,17 +244,21 @@ public static void close() {
       closeFuture.addListener(new ChannelFutureListener() {
```

Comment:
Hi!

I'm not sure what you are referring to. I didn't change anything there, that is maybe due to white-spacing, but I didn't touch that line or did not ment to touch it.